lib/api.js

```diff
@@ -59,7 +59,7 @@ var generateToken = exports.generateToken = function generateToken(config, cb) {
 
     var http_options = {
         schema: config.schema || configuration.default_options.schema,
-        host: utils.getDefaultApiEndpoint(config.mode) || config.host || configuration.default_options.host,
+        host: config.host || utils.getDefaultApiEndpoint(config.mode) || configuration.default_options.host,
```

Comment:
hmm is it a common pattern to fork service libraries in such cases?

Reply:
have you thought of making a PR for this?

Reply:
@sadappa for the sake of time I'm forking the library. @adrianlee44 yes but I'm gonna file an issue and make sure it's a regression/bug

Reply:
https://github.com/paypal/PayPal-node-SDK/issues/326